packages/consensus/src/types/utreexo.cairo

```diff
@@ -497,4 +537,40 @@ mod tests {
         assert_eq!(utreexo_state.roots, expected, "cannot add 22 leaves");
         assert_eq!(utreexo_state.num_leaves, 30);
     }
+
+    #[test]
+    fn test_utreexo_delete() {
```

Comment:
Can you please add another test for a larger tree, say 16 leaves, and you remove 3rd?
You can leverage utreexo.py for creating proofs

Reply:
done